packages/gatsby-recipes/src/recipe-machine/index.js

```diff
@@ -261,7 +262,7 @@ const recipeMachine = Machine(
         }
       }),
       addResourcesToContext: assign((context, event) => {
-        if (event.data) {
+        if (lodash.isArray(event.data) && event.data.length > 0) {
```

Comment:
`loadash.isArray` vs `Array.isArray` -> less dependencies...

Reply:
we already use lodash elsewhere